lib/Essence/Di/Container/Standard.php

```diff
@@ -434,6 +434,11 @@ protected function _setupProviders() {
 					'http://www.dipity.com/oembed/timeline?format=json&url=:url'
 				);
 			}),
+			'Documentcloud' => Container::unique(function($C) {
```

Comment:
Seems good to me, could you just indent using tabs ? 
Sorry to be picky, I just want to keep the codebase consistent ;)